src/style.css

```diff
@@ -0,0 +1,147 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background: #e5e5e5;
+}
+
+:root {
+  --fonovy-color: #fff;
+  --button-color: #00acdc;
+  --secondary_text-color: #616070;
+  --main_text-color: #060b35;
+}
+
+.conteiner {
+  width: 200px;
+  height: 408px;
+}
+
+.Product {
+  height: 406px;
+  width: 198px;
```

Comment:
you must not to fix height and width different from task
![image](https://github.com/mate-academy/layout_product-cards/assets/96265844/34cfa901-fa96-435a-844a-acc9368fcfcc)

Reply:
still not resolved, fix what mentor asked before request review

![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/30d2625b-d0be-4bd6-be32-a8135b0a08cc)